test/projects/saas-file-generate-bundle/file-generate/file-generate.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: file-generate
+channels:
+- name: staging
+  currentCSV: file-generate.v0.1.10-c664c72
```

Comment:
The test case creates a new version `0.1.6-441cd20` do we validate that this is updated?